### ISSUE TITLE
New template: Text Classification in scikit-learn [WIP]

### ISSUE THREAD

**Issue by themrcesi** [jrieke/traingenerator#4]
Hi there!

I´m planning to make a text classification template using scikit-learn. Just let me know whether this task adapts to the goal of the project.

Once I get the ok, I´ll start working on it.

**Comment by jrieke** [comment]
That is perfectly in line with the goal and as far as I can see, nobody else is working on this yet! 🎈 You can probably start off the [existing image classification template for scikit-learn](https://github.com/jrieke/traingenerator/tree/main/templates/Image%20classification_scikit-learn). Let me know if you need any input or help. And feel free to open a work in progress PR as soon as you start, so I can give feedback :)

**Comment by themrcesi** [comment]
Yeah, I´ll try to follow the same structure as much as possible 😄 Thanks!
